.changeset/mighty-glasses-rhyme.md

```diff
@@ -0,0 +1,20 @@
+---
+"@razorpay/blade": major
+---
+
+fix(Alert)!: fix typo in prop `isDismissible`
+
+> **Warning**
+>
+> Breaking change. Update prop `isDismissable` to `isDismissible`.
```

Comment:
are we sure to put it in the major release? what does the amplitude data says on the usage of this prop?

Reply:
let's club it with the other alert changes?

Reply:
There are [few occurrences](https://cs.github.com/?scope=org%3Arazorpay&scopeName=razorpay&q=isdismissable) across projects (total 9 occurrences). I think it might be better to push this in the next major release because we don't need this urgently.

Reply:
I checked the files there are like 8 occurrences. and most of them are explicitly setting this prop to false which by default is false only right? still unsure if this requires a major breaking release from political standpoint 🙊

Reply:
Default value is `true` 😅